recipe/meta.yaml

```
@@ -1,43 +1,55 @@
 {% set name = "cmarkgfm" %}
-{% set version = "0.8.0" %}
+{% set version = "0.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: fe859b0e98d222b4b458022007daac2757e970f0671d45e55d01931c1b4bb3cb
+  sha256: f20900f16377f2109783ae9348d34bc80530808439591c3d3df73d5c7ef1a00c
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv "
```

do we need `--ignore-installed`?

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv --ignore-installed
```

---

I second the question

---

also, `vv` instead of `vvv` (already in the suggestion)